packages/v4/patternfly-docs/content/contribute/about/about.md

```diff
@@ -5,24 +5,20 @@ section: contribute
 
 The PatternFly open source community depends on contributions to help our design system grow and evolve. We encourage everyone, regardless of background, to make suggestions for enhancements, contribute new design patterns and ideas, help identify bugs in code, and more. With your help, we can stay on top of the latest and greatest implementation solutions.  
 
+## PatternFly feature lifecycle
+
+PatternFly features go through a life-cycle development process that requires contributions from multiple disciplines including design, development, and content authoring. The typical feature lifecycle is illustrated below.
+
+![Contribution guide](./about-flowchart.png)
+
 ## PatternFly on GitHub
 PatternFly has a few repos you can contribute to:
 - [patternfly](https://github.com/patternfly/patternfly): For core HTML and CSS contributions. All component contributions should start in Core.
```

Comment:
```suggestion
- [patternfly](https://github.com/patternfly/patternfly) ("Core"): For core HTML and CSS contributions. All component contributions should start in Core.
```

This is just one approach for doing this, but I think it would be helpful to make it a little more explicit that this repo is called "Core" than it currently sounds.